test/jasmine/tests/gl3d_plot_interact_test.js

```diff
@@ -14,132 +14,6 @@ var drag = require('../assets/drag');
 var selectButton = require('../assets/modebar_button');
 var delay = require('../assets/delay');
 
-describe('Test gl3d before/after plot', function() {
-    var gd;
-
-    var mock = require('../../image/mocks/gl3d_marker-arrays.json');
-
-    beforeEach(function() {
-        jasmine.DEFAULT_TIMEOUT_INTERVAL = 8000;
-    });
-
-    afterEach(function() {
-        Plotly.purge(gd);
-        destroyGraphDiv();
-    });
-
-    it('@noCI @gl should not rotate camera on the very first click before scene is complete and then should rotate', function(done) {
-        var _mock = Lib.extendDeep(
-            {
-                layout: {
-                    scene: {
-                        camera: {
-                            up: {
-                                x: 0,
-                                y: 0,
-                                z: 1
-                            },
-                            eye: {
-                                x: 1.2,
-                                y: 1.2,
-                                z: 1.2
-                            },
-                            center: {
-                                x: 0,
-                                y: 0,
-                                z: 0
-                            }
-                        }
-                    }
-                }
-            },
-            mock
-        );
-
-        var x = 605;
-        var y = 271;
-
-        function _stayThere() {
-            mouseEvent('mousemove', x, y);
-            return delay(20)();
-        }
-
-        function _clickThere() {
-            mouseEvent('mouseover', x, y, {buttons: 1});
-            return delay(20)();
-        }
-
-        function _clickOtherplace() {
-            mouseEvent('mouseover', 300, 300, {buttons: 1});
-            return delay(20)();
-        }
-
-        _stayThere()
-        .then(function() {
-            gd = createGraphDiv();
-            return Plotly.newPlot(gd, _mock);
-        })
-        .then(delay(100))
```

Comment:
Thank you @emilykl for the PR.
Instead of removing these tests, could you adjust the delay after the first plot? Also you may be able to remove `@noCI` tag and it start working on the CI.